samples/powerpoint/basics/get-slide-metadata.yaml

```diff
@@ -0,0 +1,56 @@
+id: ''
+gist: ''
+source: POWERPOINT
+author: Microsoft
+name: get-slide-metadata
+description: 'Gets the title, index, and ID of the selected slide(s).'
+script:
+  content: |-
+    $("#run").click(run);
+
+    function run() {
+        Office.context.document.getSelectedDataAsync(Office.CoercionType.SlideRange,
+            function (asyncResult) {
+                if (asyncResult.status === Office.AsyncResultStatus.Failed) {
+                    console.log(asyncResult.error.message);
+                } else {                
+                    document.getElementById('output').innerText = JSON.stringify(asyncResult.value);
+                }
+            }
+        );
+    }
+  language: typescript
+style:
+  content: "body {\r\n    margin: 0;\r\n    padding: 10px;\r\n}\r\n\r\n\r\n/* Button customization, including overwriting some Fabric defaults */\r\n\r\n.ms-Button, .ms-Button:focus {\r\n    background: #b7472a;\r\n    border: #b7472a;\r\n}\r\n\r\n    .ms-Button > .ms-Button-label,\r\n    .ms-Button:focus > .ms-Button-label,\r\n    .ms-Button:hover > .ms-Button-label {\r\n        color: white;\r\n    }\r\n\r\n    .ms-Button:hover, .ms-Button:active {\r\n        background: #8e3720;\r\n    }\r\n\r\n    .ms-Button.is-disabled, .ms-Button:disabled {\r\n        background-color: #f4f4f4;\r\n        border-color: #f4f4f4;\r\n    }\r\n\r\n    .ms-Button.is-disabled .ms-Button-label,\r\n    .ms-Button:disabled .ms-Button-label {\r\n        color: #a6a6a6;\r\n    }\r\n\r\n    "
+  language: css
+template:
+  content: >
+    <p class="ms-font-m">Select one or more slides and click <b>Run</b> to get
```

Comment:
Users need to be told to select one or more slides. Also, your wording just repeats what the title says, so it seems redundant.